fmcdaq3/fmcdaq3.c

```diff
@@ -129,18 +129,15 @@ int main(void)
 	ad9528_spi_param.type = ZYNQ_PS7_SPI;
 	ad9152_spi_param.type = ZYNQ_PS7_SPI;
 	ad9680_spi_param.type = ZYNQ_PS7_SPI;
-#endif
```

Comment:
actually this change is not quite without any functional changes.
there is a semantic difference between   `#if` and `#ifdef` (which is short for `#if defined()`)
in many cases you don't notice the difference, because typically people do `#define ZYNQ_PS7   1` ; this evaluates true for `#if` and `#ifdef`, while `#define ZYNQ_PS7`  would evaluate true only for `#ifdef ZYNQ_PS7`

this PR is pretty big as is;
i would recommend we defer this change for another PR/changeset; and we can review it there